Resources/Prototypes/_SpaceCats/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```
@@ -52,3 +52,53 @@
+  - type: MagazineVisuals
+    magState: mag
+    steps: 5
+    zeroVisible: true
+  - type: Appearance
+  - type: GenericVisualizer
+    visuals:
+      enum.EnergyGunFireModeVisuals.State:
+        Firemode:
+          Disabler: { state: mode-disabler }
+          Lethal: { state: mode-lethal }
+          Special: { state: mode-stun }
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)